src/test/postFetch.jsx

```diff
@@ -0,0 +1,24 @@
+import { signInAction } from "../../src/redux/actions/loginActions";
+
+export function postFetch(login, pass, navigate) {
```

Comment:
feel free to name functions based on their actions, not based on implementation
So `login` or `loginUser` would be a better name